renderdoc/driver/vulkan/wrappers/vk_device_funcs.cpp

```diff
@@ -22,6 +22,7 @@
  * THE SOFTWARE.
  ******************************************************************************/
 
+#include "../../../driver/ihv/amd/official/GPUPerfAPI/Include/GPUPerfAPI-VK.h"
```

Comment:
I don't really want these ihv-particular includes to leak out into the rest of the codebase, I already want to refactor to remove the one in `vk_debug.cpp` which shouldn't really need to be there. If the extension isn't going to be published and make it into the vulkan headers, can you remove this and just use `"VK_AMD_gpa_interface"` instead of `VK_AMD_GPA_INTERFACE_EXTENSION_NAME`

Reply:
We were skeptical about this too, thought to add it and get your feedback; I have removed this now.